styles/nixos-manual/catppuccin.user.css

```
@@ -0,0 +1,200 @@
+/* ==UserStyle==
+@name NixOS Manual Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/nixos-manual
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/nixos-manual
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/nixos-manual/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Anixos-manual
+@description Soothing pastel theme for NixOS Manual
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document url-prefix('https://nixos.org/manual/')
```

Might as well add this one too here I think.
```suggestion
@-moz-document url-prefix('https://nixos.org/manual/'), url-prefix('https://daiderd.com/nix-darwin/manual/')
```

---

I agree